---
layout: single
title:  "Coronavirus"
categories: research-1
tag: [research, WGS, NGS, coronavirus]
toc: true
author_profile: true
typora-root-url: ../
---

# My Research Area 

Let's me explain my research area

## Coronavirus 
Coronaviruses have had a major impact on mankind since the mid-1960s. They are classified into four distinct genera: alpha, beta, gamma, and delta. Both alpha and betacoronaviruses are known to cause diseases in humans.

<br/>

### *Betacorinavirus*

---

####  SARS-CoV-1 & SARS-CoV-2 ; Clade B
Severe acute respiratory syndrome coronavirus(SARS-CoV) is a species belonging to the genus beta coronavirus of the family Coronaviridae under Nidovirales (order). There are some species. The representitive species is SARS-CoV-1 and SARS-CoV-2(so called **COVID-19** ). 

A major concern in COVID-19 outbreaks is the rapid mutation of SARS-CoV-2 viruses. The current variant typing of SARS-CoV-2 solely depends on whole genome sequencing (WGS). Thus, rapid and accurate genome sequencing of the virus is essential. In this study, we aimed to develop a rapid WGS protocol using universal primers spanning the entire genome of SARS-CoV-2. We have designed 11 pairs of SARS-CoV-2 universal primers which cover 7 genome clades (Clade of S, L, V, G, GR, GH, and GV) and 2 variants (B1.1.7 and B.1.135). The 9 viral strain’s RNAs extracted from cell cultures were used as template of PCR. The 3 kb amplicons were synthesized from the 11 primer pairs. The MinION sequencing reads were assembled into a complete genome with accuracy of 99.94-99.98%. Finally, the developed protocol was applied to five clinical samples. The results showed that the complete genome sequencing and variant typing could be directly obtained from clinical samples. Consequently, this method would contribute to rapid COVID-19 diagnosis, particularly in public health surveillance. 

![2022KOGO_KKW_poster](/assets/images/23-02-10-Coronavirus/2022KOGO_KKW_poster.jpg)

####  MERS-CoV; Clade C
Midlle East Respiratory Syndrome Coronavirus (MERS-CoV) is a species belonging to the genus beta coronavirus of the family Coronaviridae under Nidovirales (order). It has the largest genome size among RNA viruses with about 30Kb length and has positive-sense single-stranded RNA genome.

Full genome sequencing is essential to verify any variations of viral pathogens. Due to the low viral titers in clinical samples, universal primers are needed that allows amplification of a tiny amount of viral nucleic acid to be deep-sequenced. In this study, we aimed to produce a universal primer set for the genomic amplification of MERS-CoV. As a result, 13 pairs of primers were designed to produce amplicons (length of 2.5-3 kb) stretched over the entire genome and the experimental procedure was optimized. The selected primer set was applied to MERS-CoV and NeoCoV and produced amplicons with expected size. Further experiments with animal samples were also completed using the same methods. The mixture of amplicons was successfully sequenced using MinION, Miseq, Pacbio RSII, and Sanger sequencing. The sequencing result demonstrated that primer sets developed in this study have universality and specificity to MERS viruses. 

![KRICT_CVRL_김관우](/assets/images/23-02-10-Coronavirus/KRICT_CVRL_김관우.jpg)

#### OC43-CoV; Clade A

![2022 KOGO_Betacoronavirus_김관우](/assets/images/23-02-10-Coronavirus/2022 KOGO_Betacoronavirus_김관우.jpg)

<br/>


### *Alphacoronavirus*

---

alphacoronaviruses have received relatively little attention due to their low fatality rate (Vassilara et al., 2018). Two types of alphacoronaviruses infect humans: HCoV-229E, which was first described in the mid-1960s, and HCoV-NL63, which was identified in 2004. These two types of alphacoronaviruses are known to cause 30% of common colds in humans (Su et al., 2016).

![ASM_190619_smchoi](/assets/images/23-02-10-Coronavirus/ASM_190619_smchoi.jpg)

<br/>
